verilog/tools/ls/verilog-language-server_test.cc

```diff
@@ -0,0 +1,488 @@
+// Copyright 2017-2020 The Verible Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "verilog/tools/ls/verilog-language-server.h"
+
+#include "gtest/gtest.h"
+
+namespace verilog {
+namespace {
+
+// TODO (glatosinski) for JSON messages use types defined in lsp-protocol.h
+
+using namespace nlohmann;
+
+class VerilogLanguageServerTest : public ::testing::Test {
+ public:
+  // Wraps request for the Language Server in RPC header
+  void SetRequest(absl::string_view request) {
```

Comment:
Since `SetRequest()` is now only needed inside `SendRequest()`, we can make it private.

Reply:
I made those functions private